test/integration/messaging.js

```diff
@@ -27,25 +27,29 @@ describe('messaging', function() {
   });
 
   it('updates a message', function(done) {
-    senderModel.update({topic: 'public', message: 'An update message'},
-    function(err) {
-      if (err) return done(err);
-      receiverModel.find(0, function(msg) {
-        expect(msg).to.equal('An update message');
-        done();
+    setTimeout(function() {
+      senderModel.update({topic: 'public', message: 'An update message'},
+      function(err) {
+        if (err) return done(err);
+        receiverModel.find(0, function(msg) {
+          expect(msg).to.equal('An update message');
+          done();
+        });
       });
-    });
+    }, 5000);
```

Comment:
The `setTimeout()` I've added is to make the last 2 tests to have some time interval when executing.  Without that, sometimes the test failed when calling `receiverModel.find()` due to timeout.  I've also tried to make sure the teardown is done but it doesn't help.  

In my local environment, I can run it without such function.  But if running CI locally or remotely, this problem occurs.  

I understand it may not be the best solution, but I'm running out of ideas. ;)  If you have any suggestions, pls let me know!